packages/components/src/validated-form-controls/components/stories/overview.story.tsx

```diff
@@ -200,6 +196,86 @@ export const AsyncValidation: StoryObj< typeof ValidatedInputControl > = {
 			/>
 		);
 	},
+	play: async ( { canvasElement } ) => {
```

Comment:
This is nice and weird 😅  The first time I encountered I felt it was a bug. It'd be nice to trigger this intentionally from a button instead of executing it on load.

It makes it impossible to test/learn how this component works: for example, due to having this setup, the first user validation will trigger on change, not on blur. So I think I'd rather not have this if this can't be triggered by users.

Reply:
That's fair 😅 Unfortunately there's no way to disable the autoplay on the individual story pages. I'll keep the story unexported so it's still available for test purposes (Jest was a bit unreliable for this one).